src/blog/2022/08/flowforge-08-released.md

```diff
@@ -0,0 +1,71 @@
+For local installs of FlowForge, the nodes are only available with an Enterprise Edition license.
+![](../images/ProjectLink.gif)
+
+
+[Start & Stop Flows](https://github.com/flowforge/flowforge/issues/839)
+Node-RED 3.0 [introduced a new feature](https://nodered.org/blog/2022/07/14/version-3-0-released#editing-stopped-flows) that allows you to stop your flows from processing requests while still being able to work in the editor and deploy changes. We've now enabled this feature within FlowForge for projects running a Node-RED 3.x stack.
+
+[Default Team](https://github.com/flowforge/flowforge/issues/298)
+If you are a member of multiple teams you can now set your preferred default saving you from having to change teams each time you log in.
+
+## Improvements
+We've made a number of improvements to the overall experience of running FlowForge.
+
+- Devices now communicate to the Forge application over MQTT instead of polling [#754](https://github.com/flowforge/flowforge/issues/754). You'll need to update your Device Agent to the latest version to take advantage of this.
+- The table views have had a major overhaul allowing you to sort and search items [#28](https://github.com/flowforge/forge-ui-components/issues/28)
+- If the application receives an error you now see a notification in the UI. [#771](https://github.com/flowforge/flowforge/issues/771)
+- The Verification email page has been cleaned up [#718](https://github.com/flowforge/flowforge/issues/718)
+- The initial Thank-you page has been cleaned up [#648](https://github.com/flowforge/flowforge/issues/648)
+
+
+## Bug Fixes
+We've fixed the following bugs in this release.
+- [Logo Distorted in Safari](https://github.com/flowforge/flowforge/issues/793)<br>
+- [LocalFS Install doesn't check for Build Tools](https://github.com/flowforge/flowforge/issues/729)<br>
+- [Users with Expired passwords can create teams](https://github.com/flowforge/flowforge/pull/842)<br>
+- [Click-jacking Vulnerability](https://github.com/flowforge/flowforge/pull/790)
+- [Users with can create teams without verifying email](https://github.com/flowforge/flowforge/pull/824)<br>
+- [Occasional Timeout when deploying flows](https://github.com/flowforge/flowforge-nr-storage/issues/17)<br>
+- [Notification of member deletion contains internal ID](https://github.com/flowforge/flowforge/issues/833)<br>
+- [Pressing Enter in the Team Delete modal triggers cancel](https://github.com/flowforge/flowforge/issues/334)<br>
+- [Node-Red Isn't ready when Forge app says it is running (Docker)](https://github.com/flowforge/flowforge/issues/751)<br>
+
+### Upgrading FlowForge
+
+[FlowForge Cloud](https://app.flowforge.com) is already running 0.8 and the stacks updated. Upgrade your project stacks to the latest version and start using the Project Link nodes now.
```

Comment:
```suggestion
[FlowForge Cloud](https://app.flowforge.com) is already running 0.8 and the stacks are updated. [Upgrade your project stacks](link-to-docs-on-how-to) to the latest version and start using the Project Link nodes now.
```

Reply:
There aren't specific docs for this use case as its just a one click in the forge application, our stacks documentation is for admins on creating a stack.